test/Microsoft.ML.TestFramework/EnvironmentExtensions.cs

```diff
@@ -17,6 +18,7 @@ public static class EnvironmentExtensions
         public static TEnvironment AddStandardComponents<TEnvironment>(this TEnvironment env)
             where TEnvironment : IHostEnvironment
         {
+            env.ComponentCatalog.RegisterAssembly(typeof(MatrixFactorizationPredictor).Assembly); // ML.Recommender
```

Comment:
Since the `MatrixFactorizationPredictor` isn't part of the `Microsoft.ML` nuget package, I don't think this should be added by `AddStandardComponents`. This method should only add the components from the `Microsoft.ML` nuget package.

Which tests exactly need to use the ComponentCatalog to create instances of `MatrixFactorization` components? If you are using the direct access API, you shouldn't need the ComponentCatalog.  If you are using EntryPoints, then in those tests you can explicitly register the assembly, if necessary.

Reply:
Ok. Just got it removed. Thanks.